packages/bff/src/model/domain/errors.ts

```diff
@@ -2,34 +2,39 @@ import { constants } from "node:http2";
 import {
   ApiError,
   AttributeId,
+  parseErrorMessage,
   makeApiProblemBuilder,
 } from "pagopa-interop-models";
 
 export const errorCodes = {
```

Comment:
I would add the new codes at the end of the list with progressive numbers, without modifying the rest of the error codes